server/tests/participant-status.test.js

```diff
@@ -392,6 +396,66 @@ describe('Test Participant status data model and service', () => {
     expect(resultNoReject.data.length).toBe(0);
   });
 
+  it('should hide status for user', async () => {
+    const participant = await makeTestParticipant({
+      emailAddress: 'test.site.participant.5@hcap.io',
+    });
+
+    const site1 = await makeTestSite({
+      siteId: 202205270022,
```

Comment:
Is this SiteId random or does it reference a specific site?

Reply:
This is created by me by putting date as readable int format YYYYMMDDHHMM